parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -418,6 +418,115 @@ write_files:
     iptables -I FORWARD -d 168.63.129.16 -p tcp --dport 80 -j DROP
     #EOF
 
+- path: /etc/sysctl.d/999-sysctl-aks.conf
+  permissions: "0644"
+  owner: root
+  content: |
+    # This is a partial workaround to this upstream Kubernetes issue: */}}
+    # https://github.com/kubernetes/kubernetes/issues/41916#issuecomment-312428731 */}}
```

Comment:
remove the extra */}}, not sure if this will cause arm template expression parsing issue